CONTRIBUTING.md

```diff
@@ -46,23 +47,23 @@ your interactions with the project and its participants.
 ## Observability Pack Guidelines
 
 We encourage all contributors to actively engage in the creation and maintenance of
-Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+quick starts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```

Comment:
I think @nr-kkenney mentioned this in slack, but this should be `quickstarts` not `quick starts`

Reply:
```suggestion
quickstarts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```